spec/expression.dd

```diff
@@ -11,8 +11,9 @@ $(HEADERNAV_TOC)
 
 $(H2 $(LNAME2 order-of-evaluation, Order Of Evaluation))
 
-$(P Binary expressions and function arguments are evaluated in strictly
-left-to-right order.)
+$(P Binary expressions are evaluated in strictly left-to-right order.
```

Comment:
Don't think you need the dashes, `left to right`.

Reply:
I like the hyphens of the original, reads a little easier.